packages/govuk-frontend-review/src/views/full-page-examples/travel-guidance/index.njk

```diff
@@ -0,0 +1,396 @@
+---
+scenario: You want to read guidance about the entry requirements for travelling to Narnia.
+notes: Based on https://www.gov.uk/foreign-travel-advice/iceland/entry-requirements
+---
+{% extends "layouts/full-page-example.njk" %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/pagination/macro.njk" import govukPagination %}
+
+{% set pageTitle = "Entry requirements - Narnia travel advice" %}
+{% block pageTitle %}{{ pageTitle }} - GOV.UK{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: "#/"
+      },
+      {
+        text: "Passports, travel and living abroad",
+        href: "#/"
+      },
+      {
+        text: "Travel abroad",
+        href: "#/"
+      },
+      {
+        text: "Foreign travel advice",
+        href: "#/"
+      }
+    ]
+  }) }}
+{% endblock %}
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+
+    <div class="govuk-!-margin-bottom-8">
+      <span class="govuk-caption-xl">Foreign travel advice</span>
+      <h1 class="govuk-heading-xl">Narnia entry requirements</h1>
+    </div>
+    
+    <aside class="govuk-!-padding-bottom-3" role="complementary">
+      <nav aria-label="Travel advice pages" class="gem-c-contents-list" role="navigation">
+        <h2 class="gem-c-contents-list__title">Contents</h2>
+        <ol class="gem-c-contents-list__list">
+          <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--dashed">
+            <a class="gem-c-contents-list__link govuk-link" href="#/">Warnings and insurance</a>
```

Comment:
I hadn't realised we use this fake link pattern

But the other examples all populate a fake path too

```suggestion
            <a class="gem-c-contents-list__link govuk-link" href="#/warnings-and-insurance">Warnings and insurance</a>
```